### ISSUE TITLE
Update extention in addons.mozilla.org

### ISSUE THREAD

**Issue by andromeda951** [ldenoue/yawas#25]
In manifest.json the version is 7.4.1 but in addons.mozilla.org the version is 7.3.9

I can't get last update, I mean is update color yellow highlight to rgb(255, 255, 102).

Thank you.

![image](https://github.com/ldenoue/yawas/assets/85559121/74fdc7d2-c95e-4b8c-a2fb-cceece096201)
![image](https://github.com/ldenoue/yawas/assets/85559121/204cb5a5-af3c-4c29-8fc6-90941ff59450)

**Comment by ldenoue** [comment]
can you use github and install from source instead?

**Comment by andromeda951** [comment]
yes, I had install in my local computer with struggle in advance

I had modified manifest.json to version 2 because my browser can not support version 3, using firefox-nightly (developer edition) to run xpi extension without verified from addons.mozilla.org for load permanent this extension, and using this tool [mozilla/web-ext](https://github.com/mozilla/web-ext) to create xpi extension correctly

Thank in advance for this extension
